core/src/main/java/io/questdb/griffin/engine/functions/regex/NotMatchStrFunctionFactory.java

```diff
@@ -59,22 +60,22 @@ public Function newInstance(
         CharSequence regex = args.getQuick(1).getStrA(null);
 
         if (regex == null) {
-            throw SqlException.$(argPositions.getQuick(1), "NULL regex");
```

Comment:
duckdb allows `null` in regex and doesn't error out unlike us. I aligned our behaviour with duck.